bazel/repository_locations.bzl

```diff
@@ -44,11 +44,11 @@ REPOSITORY_LOCATIONS = dict(
         remote = "https://github.com/gcovr/gcovr",
     ),
     com_github_google_libprotobuf_mutator = dict(
-        commit = "c3d2faf04a1070b0b852b0efdef81e1a81ba925e",
+        commit = "f2eac713f65dc55d663426647a40aff72e68a430",  #2018-08-24
```

Comment:
I think the bumps to latest released version is a no brainer. I think for non-releases, I'm wondering if we should always be bumping without having a functional need? On the one hand, I can see the rational to be up-to-date security wise, but if we are picking random master HEAD, we might also be introducing instability etc.

Reply:
There are a bunch of dependencies that don't have releases that we can pin to and require reference by commit sha...
- abseil-cpp
- boringssl
- googleapis
- grpc-httpjson-transcoding
- jwt_verify_lib
- libcircllhist
- libprotobuf-mutator

The following can be converted from a commit sha to a release binary. Is this the preferred method?
- gcovr
- googletest
- grpc
- http_parser (currently pointing to a specific non-release commit)
- jinja
- lightstep-tracer-cpp
- markupsafe
- opentracing-cpp
- protobuf (currently pointing to a specific non-release commit)
- rapidjson
- subpar
- tclap
- xxHash

Currently using release binary:
- fmt
- spdlog
- all thrift dependencies

Reply:
wouldn't release binaries be a bad thing for architectures that aren't supported?

Reply:
This is poor terminology on my part. I'm referring to a source tar.gz artifact which is then compiled. So this isn't architecture specific.

Reply:
@moderation thanks for the analysis here. As discussed offline, the most useful, least perturbing thing to do is just convert everything consuming at a point release to use tags and bump the tags where appropriate.

For the other cases, where we use SHA, I think we can discuss on a case-by-case basis once we've done the above step, which will be a huge win for maintainability 👍

Reply:
Replacement PR https://github.com/envoyproxy/envoy/pull/4353